syntaxes/ocamldoc.json

```diff
@@ -13,6 +13,36 @@
   "repository": {
     "markup": {
       "patterns": [
+        {
+          "comment": "ocamldoc documentation tag @raise",
+          "match": "(\\@raises?)\\s+(\\S+)",
```

Comment:
I think escaping the `@` character is actually unnecessary. You can correct the old pattern on line 49 too.

Reply:
Done in [164a490](https://github.com/ocamllabs/vscode-ocaml-platform/pull/1365/commits/164a4901e589085273adbc9447e5cb69fc525314)